src/js/azdata/view/azDataGraph.js

```diff
@@ -118,7 +118,19 @@ azdataGraph.prototype.getStyledTooltipForCell = function (cell) {
 
         // tooltip heading for vertices only
         if (!cell.edge) {
-            tooltip += `<div style=\"${centerText}\"><span style=\"${boldText}\">${cell.value.label}</span></div>`;
+            let costFreeLabel = '';
+            let hasWindowsEOL = cell.value.label.includes('\r\n');
```

Comment:
I think we should just have a different field 'name' to show that over here. Either 'name' or 'tooltipTilte'